src/schema/posts.ts

```diff
@@ -115,6 +116,8 @@ export interface GQLSourcePostModeration {
   status: SourcePostModerationStatus;
   createdAt: Date;
   updatedAt: Date;
+  source: Source;
```

Comment:
source would also be optional?

Reply:
Source is always there compared to post, if I'm not mistaken.

Reply:
You are right, then in schema it should be https://github.com/dailydotdev/daily-api/pull/2449#discussion_r1842118045

Reply:
@AmarTrebinjac also you can revert https://github.com/dailydotdev/daily-api/pull/2449/commits/21c0e6ff8dad063bca27dc704a414b58ee3571ef, sorry for the juggle 😆

Reply:
@AmarTrebinjac haha, yep. I did a double take and checked, but my local branch is memeing now 💩💩